### PR TITLE
feat: add custom management console URL handling

### PR DESCRIPTION
**Pull request by jbouchery**
resolves #4739

### CODE REVIEW

File: apim/3.x/templates/api/api-configmap.yaml [hunk 1]
```diff
@@ -34,6 +34,9 @@ data:
     {{- end }}
 
     management:
+      {{- if .Values.ui.ingress.enabled }}
+      url: https://{{ index .Values.ui.ingress.hosts 0 }}{{ regexFind "/[a-zA-Z]+" .Values.ui.ingress.path }}/
```
Comment (brasseld) [inline apim/3.x/templates/api/api-configmap.yaml:38]: what is the purpose of the regexFind here ?

Comment (jbouchery) [reply apim/3.x/templates/api/api-configmap.yaml:38]: By default, the path is /console(/|$)(.*) and this regexFind keep the /console part (Keep the "/" and n letters)

Maybe i must also add numbers.

Comment (brasseld) [reply apim/3.x/templates/api/api-configmap.yaml:38]: Can't we simply keep `.Values.ui.ingress.path` only without the trailing slash ?

Comment (jbouchery) [reply apim/3.x/templates/api/api-configmap.yaml:38]: Yes but actually the default ingress rule doesn't to work without the trailing slash, even if i use this configuration : https://kubernetes.github.io/ingress-nginx/examples/rewrite/#rewrite-target

I have to type `https://my.awesome.url/console/` to work, without the trailing slash every call go to  `https://my.awesome.url/` (so JS and CSS calls don't go to my console pod)

Comment (brasseld) [reply apim/3.x/templates/api/api-configmap.yaml:38]: Yes, I still don't understand this, and had no time to waste on it.

**Pull request merged**